__tests__/integration-webhooks.test.js

```diff
@@ -99,14 +82,17 @@ describe('Metrics SDK Webhook Integration Tests', () => {
   });
 
   it('should return with a 401 if the signature is empty/missing', async () => {
-    const response = await post(`http://localhost:${PORT}/webhook`, JSON.stringify({ email: 'dom@readme.io' }), {
+    const response = await fetch(`http://localhost:${PORT}/webhook`, {
```

Comment:
Switching to using fetch()... that's fair, i didnt really like my hacky http promise extensions 😂. I think the intention was to keep it fairly simple, but i think we're past that now.